### ISSUE TITLE
Failed to Compile due to babel-loaders

### ISSUE THREAD

**Issue by joaortizro** [concludio/react-mathlive#36]
when installing on a new react project , babel loaders suggest to add extra configuration.

![Screenshot_20210806_154750](https://user-images.githubusercontent.com/20843665/128569627-b9b0f48e-f084-4171-b04b-224aa64ef6c7.png)

**Comment by joaortizro** [comment]
anyone having the same problems?

**Comment by arnog** [comment]
This is a duplicate of https://github.com/arnog/mathlive/issues/1123

**Comment by joaortizro** [comment]
@arnog it seems it's a problem with the react-scripts versions, i've tried 3.3.0,3.4.0 and 3.4.1  and still no success, but there was a change , now the error is on the ''??" operator, will dig futher and report any updates

**Comment by arnog** [comment]
@joaortizro yes, it's due to some tools in the react-cli toolchain still using webpack-4, which does not support the `??` operator. The fix is in the mathlive depot, which is to target `es2019` instead of the current version of Javascript. But maybe you can find a way to configure react-cli to use webpack-5.
